crates/terminal_view/src/terminal_panel.rs

```diff
@@ -196,6 +197,22 @@ impl TerminalPanel {
                                 }
                             }),
                     )
+                    .child({
+                        let layout = Workspace::is_bottom_dock_full_width(&workspace, cx);
+                        IconButton::new("toggle_extended", IconName::ExpandHorizontal)
```

Comment:
Let's make this button's icon swap between Lucide's `chevrons-left-right` and `chevrons-right-left`